cmd/tsbs_generate_queries/databases/timescaledb/devops.go

```diff
@@ -227,6 +233,7 @@ func (d *Devops) HighCPUForHosts(qi query.Query, nHosts int) {
 	d.fillInQuery(qi, humanLabel, humanDesc, sql)
 }
 
+// fill Query fills teh query struct with data
```

Comment:
`// fillInQuery fills the query struct with data`

Reply:
Yes, agree, typo